specifyweb/frontend/js_src/lib/components/FormParse/fields.ts

```diff
@@ -212,6 +213,8 @@ const processFieldType: {
         hasCloneButton: getProperty('cloneBtn')?.toLowerCase() === 'true',
         typeSearch: getProperty('name'),
         searchView: getProperty('searchView'),
+        hasAddButton:
+          (getProperty('addBtn')?.toLowerCase() ?? 'true') === 'true',
```

Comment:
Can be simplified:
```ts
getProperty('addBtn')?.toLowerCase() !== 'false'
```

Plus, if user makes a typo and says `true ` or `truee`, your version would default to `false`, where as this version would default to `true`